src/Maker/MakeEntity.php

```diff
@@ -160,6 +163,12 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             'Entity\\'
         );
 
+        $mappingDriver = $this->doctrineHelper->getMappingDriverForNamespace($entityClassDetails->getFullName());
+
+        if ($mappingDriver instanceof AttributeDriver && !$this->doctrineHelper->canUseAttributes()) {
+            throw new RuntimeCommandException('To use attributes you\'ll need atleast PHP 8, Doctrine 2.9, and Symfony 5.2.');
```

Comment:
```suggestion
            throw new RuntimeCommandException('To use attributes you\'ll need at least PHP 8, Doctrine 2.9, and Symfony 5.2.');
```

Also, you need `doctrine/orm` 2.9, right? And also `doctrine/doctrine-bundle` at some version?

Reply:
I'd say, let's be specific! Something like:

> To use Doctrine entity attributes, you'll need PHP 8, doctrine/orm 2.9... doctrine/doctrine-bundle X.? and symfony/framework-bundle 5.2

Notice the doctrine-bundle version if `X.?` - so don't just copy and paste it 😉

Reply:
works for me!